azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/features/VirtualMachineScaleSetApiLiveTest.java

```diff
@@ -0,0 +1,267 @@
+
+        return VirtualMachineScaleSetOSProfile.create(vmssName, "jclouds", "jClouds1!",
+                linuxConfiguration, windowsConfiguration, null);
+    }
+
+
+    private VirtualMachineScaleSetNetworkProfile getNetworkProfile() {
+        List<NetworkProfile.NetworkInterface> networkInterfacesList = new ArrayList<NetworkProfile.NetworkInterface>();
+
+        NetworkInterfaceCard nic = createNetworkInterfaceCard(resourceGroupName, "jc-nic-" + RAND, LOCATION, "ipConfig-" + RAND);
+        assertNotNull(nic);
+        networkInterfacesList.add(NetworkProfile.NetworkInterface.create(nic.id(), NetworkProfile.NetworkInterface.NetworkInterfaceProperties.create(true)));
+
+        List<NetworkInterfaceConfiguration> networkInterfaceConfigurations = new ArrayList<NetworkInterfaceConfiguration>();
+        List<VirtualMachineScaleSetIpConfiguration> virtualMachineScaleSetIpConfigurations = new ArrayList<VirtualMachineScaleSetIpConfiguration>();
+
+
+        VirtualMachineScaleSetPublicIPAddressConfiguration publicIPAddressConfiguration =
+                VirtualMachineScaleSetPublicIPAddressConfiguration.create("pub1", VirtualMachineScaleSetPublicIPAddressProperties.create(15));
+
+
+        VirtualMachineScaleSetIpConfigurationProperties virtualMachineScaleSetIpConfigurationProperties =
+                VirtualMachineScaleSetIpConfigurationProperties.create(publicIPAddressConfiguration,
+                        this.subnet, "IPv4", null,
+                        null, null);
+
+        VirtualMachineScaleSetIpConfiguration virtualMachineScaleSetIpConfiguration =
+                VirtualMachineScaleSetIpConfiguration.create("ipconfig1", virtualMachineScaleSetIpConfigurationProperties);
+
+        virtualMachineScaleSetIpConfigurations.add(virtualMachineScaleSetIpConfiguration);
+
+        VirtualMachineScaleSetNetworkSecurityGroup networkSecurityGroup = null;
+
+        ArrayList<String> dnsList = new ArrayList<String>();
+        dnsList.add("8.8.8.8");
+        VirtualMachineScaleSetDNSSettings dnsSettings =  VirtualMachineScaleSetDNSSettings.create(dnsList);
+
+        NetworkInterfaceConfigurationProperties networkInterfaceConfigurationProperties =
+                NetworkInterfaceConfigurationProperties.create(true, false, networkSecurityGroup, dnsSettings, virtualMachineScaleSetIpConfigurations);
+        NetworkInterfaceConfiguration networkInterfaceConfiguration = NetworkInterfaceConfiguration.create("nicconfig1", networkInterfaceConfigurationProperties);
+        networkInterfaceConfigurations.add(networkInterfaceConfiguration);
+
+        return VirtualMachineScaleSetNetworkProfile.create(networkInterfaceConfigurations);
+    }
+
+
+    private ExtensionProfile getExtensionProfile() {
+        List<Extension> extensions = new ArrayList<Extension>();
+
+        List<String> uris = new ArrayList<String>();
+        uris.add("https://mystorage1.blob.core.windows.net/winvmextekfacnt/SampleCmd_1.cmd");
```

Comment:
Is this something tied to a particular account? Will this prevent the tests from succeeding for any user?

Reply:
Good point... Do you have location to store a fixed file?  I can check this file first and call it using a RAW GitHub url inside the jclouds repo.

Reply:
It could be a file in the `src/test/resources` folder of this provider, if needed.